site/downloads.markdown

```diff
@@ -9,168 +9,92 @@ isDownloads: true
+  [Stack](https://packages.fedoraproject.org/pkgs/haskell-platform/stack/)
 
-*   [petersen/ghc-8.6.5 Copr repo (EL7)](https://copr.fedorainfracloud.org/coprs/petersen/ghc-8.6.5/)
-*   [petersen/ghc-8.4.4 Copr repo (EL7)](https://copr.fedorainfracloud.org/coprs/petersen/ghc-8.4.4/)
-*   [petersen/ghc-8.2.2 Copr repo (EL7,EL6)](https://copr.fedorainfracloud.org/coprs/petersen/ghc-8.2.2/)  
-*   [petersen/ghc-8.0.2 Copr repo (EL7)](https://copr.fedorainfracloud.org/coprs/petersen/ghc-8.0.2)
-
-Note that the different ghc package versions cannot be installed in parallel.
-
-##### <span style="text-decoration: underline">Arch Linux</span>
-
-*Note: installing the Haskell toolchain via the Arch Linux package manager is not recommended, since it enforces dynamic linking, which can cause various issues. Instead consider [GHCup](#ghcup).*
-
-The official repos on Arch Linux contain packages `ghc`, `cabal-install`, `happy`, `alex`, `haddock`. Install them with:
-
-<pre>sudo pacman -S ghc cabal-install happy alex haskell-haddock-library</pre>
- 
-##### <span style="text-decoration: underline">openSUSE Leap</span>
-
-*   Leap 15.1 has ghc-8.6.4 and cabal-install-2.4.0.0
-
-To install from official openSUSE:Leap repo, just run the install command:
-
-`sudo zypper in ghc cabal-install`
-
-For last stable version you can use the development openSUSE repository:
+#### openSUSE
 
+* Official packages
+  [GHC](https://software.opensuse.org/package/ghc?search_term=%22ghc%22),
+  [cabal](https://software.opensuse.org/package/cabal-install),
 * [devel:languages:haskell repo](https://build.opensuse.org/project/show/devel:languages:haskell)
 
-Use this command to add repository to your system:
-
-`sudo zypper ar -f -p 90 https://download.opensuse.org/repositories/devel:/languages:/haskell/openSUSE_Leap_15.1/devel:languages:haskell.repo`
-
-##### <span style="text-decoration: underline">openSUSE Tumbleweed</span>
-
-*   Tumbleweed has last stable version of ghc and cabal install.
+#### Gentoo
 
-To install from official openSUSE:Tumbleweed repo, just run the install command:
+* Official packages
+  [GHC](https://packages.gentoo.org/packages/dev-lang/ghc),
+  [cabal](https://packages.gentoo.org/packages/dev-haskell/cabal),
+  [Stack](https://packages.gentoo.org/packages/dev-haskell/stack)
+* [Third party repo](https://github.com/gentoo-haskell/gentoo-haskell)
 
-`sudo zypper in ghc cabal-install`
+#### Arch
 
-##### <span style="text-decoration: underline">Gentoo</span>
-
-While the Haskell toolchain is available in the main gentoo repository, it is recommended
-to use the more up-to-date [gentoo-haskell](https://github.com/gentoo-haskell/gentoo-haskell) overlay. This can be done using layman:
-
-```
-sudo layman -a haskell
-sudo emerge --ask dev-lang/ghc dev-haskell/cabal-install
-```
+**Do not use the Haskell development tools provided by Arch, they are
+broken.** For more information see
```

Comment:
```suggestion
**Haskell tools and libraries natively packaged by Arch do not work for local Haskell development.** For more information see
```

Reply:
Seems like we could be a bit more specific here—if I'm understanding the situation correctly, the Arch approach is fine for stuff like packaging Pandoc for Arch users, but it does not work well for *doing Haskell development in general*.

Reply:
Sure, but someone packaging Pandoc for Arch users already knows that, right?  The ratio among visitors to this page of  Arch users who are not trying to package for Arch : Arch developers who are trying to package for arch must be something like 1000 : 1, or more right?  In which case I think it's probably unhelpful to be more nuanced, even if it is technically correct.

Put another way, the people who would be negatively effected by the lack of nuance in the original wording already know the solution anyway, so I don't think there's much point going out of our way to accommodate them at the expense of presenting a clear message.

Reply:
We want to make *very* clear to users that they shouldn't use arch packages. The word **broken** is key here.

Reply:
Seems fine either way.